src/Chr.Avro.Binary/BinaryDeserializerBuilder.cs

```diff
@@ -1570,6 +1565,44 @@ public override Delegate BuildDelegate(TypeResolution resolution, Schema schema,
 
             return compiled;
         }
+               
+        /// <summary>
+        /// Whether the resolved Constructor matches a RrecordSchemas fields.
+        /// </summary>
+        /// <param name="constructor">
+        /// The ConstructorResolution to check for match.
+        /// </param>
+        /// <param name="recordFields">
+        /// The RecordSchemas fields to match the against constructors parameters. 
```

Comment:
nits: "RecordSchemas" -> "record schema"; "the against" -> "against the"; "constructors" -> "the constructor"